doc/changes/changes49.xml

```diff
@@ -0,0 +1,1209 @@
+</Item>
+<Item>
+Fix <Ref BookName="ref" Oper="DisplaySemigroup"/> for transformation semigroups
+(<URL><LinkText>&Hash;1785</LinkText><Link>https://github.com/gap-system/gap/pull/1785</Link></URL>)
+</Item>
+<Item>
+Fix <Q>no method found</Q> errors in <Ref BookName="ref" Func="MagmaWithOne"/>
+and <Ref BookName="ref" Func="MagmaWithInverses"/>
+(<URL><LinkText>&Hash;1798</LinkText><Link>https://github.com/gap-system/gap/pull/1798</Link></URL>)
+</Item>
+<Item>
+Fix an error computing kernel of group homomorphism from fp group
+into permutation group
+(<URL><LinkText>&Hash;1809</LinkText><Link>https://github.com/gap-system/gap/pull/1809</Link></URL>)
+</Item>
+<Item>
+Fix an error in MTC losing components when copying a new augmented coset table
+(<URL><LinkText>&Hash;1809</LinkText><Link>https://github.com/gap-system/gap/pull/1809</Link></URL>)
+</Item>
+<Item>
+Fix output of <Ref BookName="ref" Func="Where"/> in a break loop,
+which pointed at the wrong code line in some cases
+(<URL><LinkText>&Hash;1814</LinkText><Link>https://github.com/gap-system/gap/pull/1814</Link></URL>)
+</Item>
+<Item>
+Fix the interaction of signals in &GAP; and the <Package>IO</Package> package
+(<URL><LinkText>&Hash;1851</LinkText><Link>https://github.com/gap-system/gap/pull/1851</Link></URL>)
+</Item>
+<Item>
+Make line editing resilient to <C>LineEditKeyHandler</C> failure
+(in particular, don't crash)
+(<URL><LinkText>&Hash;1856</LinkText><Link>https://github.com/gap-system/gap/pull/1856</Link></URL>)
+</Item>
+<Item>
+Omit non-characters from <Ref BookName="ref" Func="PermChars"/> results
+(<URL><LinkText>&Hash;1867</LinkText><Link>https://github.com/gap-system/gap/pull/1867</Link></URL>)
+</Item>
+<Item>
+Fix <Ref BookName="ref" Oper="ExteriorPower"/> when exterior power
+is 0-dimensional (used to return a 1-dimensional result)
+(<URL><LinkText>&Hash;1872</LinkText><Link>https://github.com/gap-system/gap/pull/1872</Link></URL>)
+</Item>
+<Item>
+Fix recursion depth trap and other improvements for quotients of fp groups
+(<URL><LinkText>&Hash;1884</LinkText><Link>https://github.com/gap-system/gap/pull/1884</Link></URL>)
+</Item>
+<Item>
+Fix a bug in the computation of a permutation group isomorphic
+to a group of automorphisms
+(<URL><LinkText>&Hash;1907</LinkText><Link>https://github.com/gap-system/gap/pull/1907</Link></URL>)
+</Item>
+<Item>
+Fix bug in <Ref BookName="ref" Func="InstallFlushableValueFromFunction"/>
+(<URL><LinkText>&Hash;1920</LinkText><Link>https://github.com/gap-system/gap/pull/1920</Link></URL>)
+</Item>
+<Item>
+Fix <Ref BookName="ref" Attr="ONanScottType"/> and introduce
+<Ref BookName="ref" Attr="RestrictedInverseGeneralMapping"/>
+(<URL><LinkText>&Hash;1937</LinkText><Link>https://github.com/gap-system/gap/pull/1937</Link></URL>)
+</Item>
+<Item>
+Fix <Ref BookName="ref" Oper="QuotientMod"/> documentation, and the integer
+implementation. This partially reverts changes made in 2013. The documentation
```

Comment:
Instead of (or in addition to) saying 2013, let's say this was in version GAP 4.7.8, which I think is more helpful for most people.